src/TLuaInterpreter.cpp

```diff
@@ -12364,6 +12364,10 @@ void TLuaInterpreter::initLuaGlobals()
     luaL_dostring(pGlobalLua, QString("package.cpath = package.cpath .. ';%1/?.so'").arg(QCoreApplication::applicationDirPath()).toUtf8().constData());
     luaL_dostring(pGlobalLua, QString("package.path = package.path .. ';%1/?.lua'").arg(QCoreApplication::applicationDirPath()).toUtf8().constData());
 #endif
+#ifdef Q_OS_WIN32
```

Comment:
Humm, I've just spotted that this may well have conflicts with https://github.com/Mudlet/Mudlet/commit/69ab37f7019408a0bf5f963f951f11c33b0d7972 which introduced support for FreeBSD as it was started before that hit the codebase.  You might want to merge in development and resolve any conflicts before trying to rebase and merge this PR... 😟